assets/css/senseilms-licensing.scss

```diff
@@ -0,0 +1,8 @@
+#section-changelog {
+	h2 {
+		clear: none;
+	}
+	h3 {
+		margin: 0;
+	}
+}
```

Comment:
I think both won't be enqueued together because it will skip earlier in Sensei LMS if `$res` was already set.
And if for some reason it's enqueued together, it's probably not a problem, since it's the same styles.